frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -84,22 +86,30 @@ const DataConsumptionCard: React.FC<DashboardItemProps> = ({
   let maxVal: number;
   let chartData = [];
   let legendData = [];
+  let suffixLabel = '';
   const result = _.get(dataConsumptionQueryResult, 'data.result', []);
   if (result.length) {
     let maxData: BarChartData | any = {
       x: '',
       y: 0,
       name: '',
+      yOriginal: 0,
+      unit: '',
     };
     chartData = metricsChartDataMap[metricType][sortByKpi](result);
     legendData = metricsChartLegendDataMap[metricType][sortByKpi](chartData);
-    maxData = _.maxBy(chartData.map((data) => _.maxBy(data, 'y')), 'y');
-    maxVal = maxData.y;
-    maxUnit =
-      (sortByKpi === BY_IOPS || sortByKpi === BY_PHYSICAL_VS_LOGICAL_USAGE) &&
-      chartData.length === 2
-        ? humanizeDecimalBytes(maxVal).unit
-        : humanizeBinaryBytesWithoutB(maxVal).unit;
+    maxData = _.maxBy(chartData.map((data) => _.maxBy(data, 'yOriginal')), 'yOriginal');
+    const maxDataV = Number(maxData.yOriginal);
+    const maxDataVStr = maxDataV.toFixed(1);
+    let maxDataH = humanizeBinaryBytes(maxDataVStr);
+    if (sortByKpi === BY_IOPS) {
```

Comment:
not only for IOPS but for others as well the label suffixes should be shown and we must omit units in y axis, which then resolves the cutoff issues to some extent as well.

Reply:
Acknowledged...

Reply:
Some screen shots...

![image](https://user-images.githubusercontent.com/895234/63687808-33348000-c823-11e9-8cdb-b77a9a01b734.png)

![image](https://user-images.githubusercontent.com/895234/63687863-55c69900-c823-11e9-89e8-32a36088e4b1.png)